package.json

```diff
@@ -6,9 +6,9 @@
     "dev": "vite --port 9999 --host -- use-remote-models",
     "css-update": "npx tailwindcss -i ./style/tailwind-input.css -o ./style/tailwind.css --watch",
     "generate-ssl": "openssl req -newkey rsa:2048 -new -nodes -x509 -days 3650 -keyout key.pem -out cert.pem",
-    "prod:github": "npm run build:use-remote-models && node build_scripts.js build-sub-projects --github && node build_scripts.js copy-models",
-    "prod:use-remote-models": "npm run build:use-remote-models && node build_scripts.js build-sub-projects && node build_scripts.js copy-models",
-    "prod": "npm run build && node build_scripts.js fetch-models && node build_scripts.js build-sub-projects && node build_scripts.js copy-models",
+    "prod:github": "npm run build:use-remote-models && node build_scripts.js build-sub-projects --github && node build_scripts.js copy-resources --use-remote-models",
+    "prod:use-remote-models": "npm run build:use-remote-models && node build_scripts.js build-sub-projects --use-remote-models && node build_scripts.js copy-resources --use-remote-models",
```

Comment:
Can we shorten the command and wrap then into the functions of build scripts.js if they are all required? Thanks! @NingW101